tests/process.issue.test.ts

```diff
@@ -23,7 +23,7 @@ import configuration from "../src/configuration/config-reader";
 import validConfiguration from "./__mocks__/results/valid-configuration.json";
 import "../src/parser/command-line";
 
-const issueUrl = process.env.TEST_ISSUE_URL || "https://github.com/ubiquibot/comment-incentives/issues/22";
+const issueUrl = process.env.TEST_ISSUE_URL || "https://github.com/ubiquibot/conversation-rewards/issues/5";
```

Comment:
Is it okay to change? I think the mocks return `/comment-incentives/issues/22` API responses.

Reply:
Change was needed to have an issue that has PR comment code reviews so that evaluation module can be fully tested. The last one had no PR code review comments. 
But I think I missed the updating in the mock. I am going to look into it.

Reply:
I have added mocks or the new issue and its PR.